docs/reference/software-versions.mdx

```diff
@@ -25,15 +25,14 @@ Release candidates are software releases that are also released to the [Testnet]
 | Soroban Environment | `v21.0.1` |
 | Stellar Core | `v21.0.0rc1` |
 | Soroban Rust SDK | `21.0.1-preview.1` |
-| Soroban CLI | `` |
-| Soroban RPC | `` |
-| Stellar Horizon | `` |
-| Stellar Friendbot | `` |
```

Comment:
This delete is intentional. It never gets updated so I don't think it's necessary to list here

Reply:
In the quickstart image where it gets updated we also just use the Horizon version tag anyway.